README.md

```diff
@@ -56,14 +56,19 @@ Throws a `NotFoundError` if the path to the requested directory does not exist.
 ### getReadStream(path:  string):  Promise\<Webdav.Stream\>
 Gets a read stream to a remote Nextcloud `path`.
 
+### getProperties(path: string, extraProperties?: FileDetailProperty[]):  Promise\<FolderProperties\>
+Retrieves the properties for the folder.
+Use extraProperties to request properties not returned by default.
+
 ### touchFolder(path:  string):  Promise\<void\>
 Smart `mkdir` implementation that doesn't complain if the folder at `path` already exists.
 
 ### getFiles(path:  string):  Promise\<string[]\>
 List files in a directory.
 
-### getFolderFileDetails(path:  string):  Promise\<FileDetails[]\>
-Same as `getFiles`, but returns full details instead of just file names.
+### getFolderFileDetails(path:  string, extraProperties?: FileDetailProperty[]):  Promise\<FileDetails[]\>
```

Comment:
The name is not new, but a little confusing. What does `FolderFile` mean? Folder or File, or File inside Folder?

Reply:
It appears to mean "Get the file details of this Folder".

Reply:
That's quite clear to me as well, it means "file of a given folder"